# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/IndexedDB/structured-clone.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/IndexedDB/structured-clone.any.js.ini
--- a/testing/web-platform/meta/IndexedDB/structured-clone.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/IndexedDB/structured-clone.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -94,142 +94,150 @@
 
 [structured-clone.any.html?101-last]
   expected:
-    if (os == "win") and not debug and not fission and (processor == "x86_64"): ["TIMEOUT", "OK"]
-    if (os == "win") and not debug and fission: ["TIMEOUT", "OK"]
+    if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["TIMEOUT", "OK"]
+    if (os == "win") and not debug and webrender: ["TIMEOUT", "OK"]
     TIMEOUT
   [Not serializable: Window: [object Window\]]
     expected:
-      if (os == "win") and not debug and not fission and (processor == "x86_64"): ["NOTRUN", "PASS"]
-      if (os == "win") and not debug and fission: ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and webrender: ["NOTRUN", "PASS"]
       NOTRUN
 
   [Not serializable: symbol: Symbol(desc)]
     expected:
-      if (os == "win") and not debug and not fission and (processor == "x86_64"): ["NOTRUN", "FAIL"]
-      if (os == "win") and not debug and fission: ["NOTRUN", "FAIL"]
+      if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["NOTRUN", "FAIL"]
+      if (os == "win") and not debug and webrender: ["NOTRUN", "FAIL"]
       NOTRUN
 
   [FileList: [object FileList\]]
     expected:
-      if (os == "win") and not debug and not fission and (processor == "x86_64"): ["NOTRUN", "FAIL"]
-      if (os == "win") and not debug and fission: ["NOTRUN", "FAIL"]
+      if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["NOTRUN", "FAIL"]
+      if (os == "win") and not debug and webrender: ["NOTRUN", "FAIL"]
       NOTRUN
 
   [Not serializable: function: function() {}]
     expected:
-      if (os == "win") and not debug and not fission and (processor == "x86_64"): ["NOTRUN", "PASS"]
-      if (os == "win") and not debug and fission: ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and webrender: ["NOTRUN", "PASS"]
       NOTRUN
 
   [File: [object File\]]
     expected:
-      if (os == "linux") and webrender and debug and not fission and not swgl: ["NOTRUN", "TIMEOUT"]
+      if (os == "linux") and (processor == "x86_64") and not swgl and webrender and debug: ["NOTRUN", "TIMEOUT"]
+      if (os == "linux") and (processor == "x86_64") and not swgl and not webrender and not debug: ["NOTRUN", "TIMEOUT"]
       if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "linux") and (processor == "x86_64") and swgl: ["NOTRUN", "TIMEOUT"]
+      if (os == "android") and not debug and webrender: ["NOTRUN", "TIMEOUT"]
+      if (os == "android") and not debug and not webrender: ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "android") and debug and not webrender: ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "win") and webrender and not fission: ["NOTRUN", "PASS"]
       if (os == "win") and webrender and fission: ["NOTRUN", "PASS", "TIMEOUT"]
-      if (os == "win") and webrender and not fission: ["NOTRUN", "PASS"]
-      if (os == "linux") and not webrender and not debug: ["NOTRUN", "TIMEOUT"]
-      if (os == "android") and not webrender: ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "linux") and (processor == "x86"): ["NOTRUN", "TIMEOUT"]
       NOTRUN
 
   [Not serializable: Event: [object Event\]]
     expected:
-      if (os == "win") and not debug and not fission and (processor == "x86_64"): ["NOTRUN", "PASS"]
-      if (os == "win") and not debug and fission: ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and webrender: ["NOTRUN", "PASS"]
       NOTRUN
 
   [Not serializable: MessageChannel: [object MessageChannel\]]
     expected:
-      if (os == "win") and not debug and not fission and (processor == "x86_64"): ["NOTRUN", "PASS"]
-      if (os == "win") and not debug and fission: ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and not webrender and (processor == "x86_64"): ["NOTRUN", "PASS"]
+      if (os == "win") and not debug and webrender: ["NOTRUN", "PASS"]
       NOTRUN
 
   [Blob: [object Blob\]]
     expected:
-      if (os == "linux") and webrender and debug and not fission and not swgl: ["TIMEOUT", "PASS"]
-      if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["TIMEOUT", "PASS"]
-      if (os == "linux") and not webrender and not debug: ["TIMEOUT", "PASS"]
-      if (os == "win") and webrender and not fission: ["TIMEOUT", "PASS"]
-      if (os == "win") and webrender and fission: ["TIMEOUT", "PASS"]
-      if (os == "android") and not webrender: ["TIMEOUT", "PASS"]
-      TIMEOUT
+      if (os == "linux") and (processor == "x86_64") and not swgl and webrender and not debug: TIMEOUT
+      if (os == "linux") and (processor == "x86_64") and not swgl and not webrender and debug: TIMEOUT
+      if (os == "win") and not webrender and not debug and (processor == "x86"): TIMEOUT
+      if (os == "win") and not webrender and debug: TIMEOUT
+      if (os == "android") and debug and webrender: TIMEOUT
+      if os == "mac": TIMEOUT
+      [TIMEOUT, PASS]
 
 
 [structured-clone.any.worker.html?101-last]
   expected:
+    if (os == "win") and webrender and not debug and not fission: ["TIMEOUT", "OK"]
+    if (os == "win") and webrender and not debug and fission: ["TIMEOUT", "OK"]
     if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["TIMEOUT", "OK"]
+    if (os == "win") and webrender and debug: ["TIMEOUT", "OK"]
     if (os == "android") and debug and not webrender: ["TIMEOUT", "OK"]
-    if (os == "win") and webrender and swgl: ["OK", "TIMEOUT"]
-    if (os == "win") and webrender and not swgl: ["TIMEOUT", "OK"]
     if (os == "mac") and debug: ["TIMEOUT", "OK"]
     TIMEOUT
   [Not serializable: function: function() {}]
     expected:
+      if (os == "win") and webrender and not debug and not fission: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and not debug and fission: ["NOTRUN", "PASS"]
       if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "PASS"]
-      if (os == "win") and webrender and swgl: ["PASS", "NOTRUN"]
-      if (os == "win") and webrender and not swgl: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and debug: ["NOTRUN", "PASS"]
       if (os == "android") and debug and not webrender: ["NOTRUN", "PASS"]
       if (os == "mac") and debug: ["NOTRUN", "PASS"]
       NOTRUN
 
   [Not serializable: MessageChannel: [object MessageChannel\]]
     expected:
+      if (os == "win") and webrender and not debug and fission: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and not debug and not fission: ["NOTRUN", "PASS"]
       if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "PASS"]
       if (os == "android") and debug and not webrender: ["NOTRUN", "PASS"]
-      if (os == "win") and webrender and not swgl: ["NOTRUN", "PASS"]
-      if (os == "win") and webrender and swgl: ["PASS", "NOTRUN"]
+      if (os == "win") and webrender and debug: ["NOTRUN", "PASS"]
       if (os == "mac") and debug: ["NOTRUN", "PASS"]
       NOTRUN
 
   [Not serializable: symbol: Symbol(desc)]
     expected:
+      if (os == "win") and webrender and not debug and fission: ["NOTRUN", "FAIL"]
+      if (os == "win") and webrender and not debug and not fission: ["NOTRUN", "FAIL"]
       if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "FAIL"]
+      if (os == "win") and webrender and debug: ["NOTRUN", "FAIL"]
       if (os == "android") and debug and not webrender: ["NOTRUN", "FAIL"]
-      if (os == "win") and webrender and swgl: ["FAIL", "NOTRUN"]
-      if (os == "win") and webrender and not swgl: ["NOTRUN", "FAIL"]
       if (os == "mac") and debug: ["NOTRUN", "FAIL"]
       NOTRUN
 
   [File: [object File\]]
     expected:
       if (os == "linux") and webrender and debug and not fission and not swgl: ["NOTRUN", "TIMEOUT"]
-      if (os == "win") and webrender and not fission and swgl: ["PASS", "TIMEOUT", "NOTRUN"]
-      if (os == "win") and webrender and not fission and not swgl: ["NOTRUN", "PASS", "TIMEOUT"]
-      if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "TIMEOUT", "PASS"]
-      if (os == "linux") and not webrender and not debug and (processor == "x86"): ["NOTRUN", "TIMEOUT"]
-      if (os == "linux") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "TIMEOUT"]
-      if (os == "win") and webrender and fission: ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "win") and webrender and not debug and not fission: ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "win") and webrender and not debug and fission: ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "PASS", "TIMEOUT"]
       if (os == "android") and debug and not webrender: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and debug: ["NOTRUN", "PASS", "TIMEOUT"]
+      if (os == "linux") and not webrender and not debug: ["NOTRUN", "TIMEOUT"]
       if (os == "mac") and debug: ["NOTRUN", "PASS", "TIMEOUT"]
       NOTRUN
 
   [Not serializable: Event: [object Event\]]
     expected:
       if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "PASS"]
-      if (os == "win") and webrender and swgl: ["PASS", "NOTRUN"]
-      if (os == "win") and webrender and not swgl: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and not debug and not fission: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and not debug and fission: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and debug: ["NOTRUN", "PASS"]
       if (os == "android") and debug and not webrender: ["NOTRUN", "PASS"]
       if (os == "mac") and debug: ["NOTRUN", "PASS"]
       NOTRUN
 
   [Not serializable: DedicatedWorkerGlobalScope: [object DedicatedWorkerGlobalScope\]]
     expected:
+      if (os == "win") and webrender and not debug and not fission: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and not debug and fission: ["NOTRUN", "PASS"]
       if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["NOTRUN", "PASS"]
-      if (os == "win") and webrender and swgl: ["PASS", "NOTRUN"]
-      if (os == "win") and webrender and not swgl: ["NOTRUN", "PASS"]
       if (os == "android") and debug and not webrender: ["NOTRUN", "PASS"]
+      if (os == "win") and webrender and debug: ["NOTRUN", "PASS"]
       if (os == "mac") and debug: ["NOTRUN", "PASS"]
       NOTRUN
 
   [Blob: [object Blob\]]
     expected:
       if (os == "linux") and webrender and debug and not fission and not swgl: ["TIMEOUT", "PASS"]
-      if (os == "linux") and not webrender and not debug and (processor == "x86"): ["TIMEOUT", "PASS"]
-      if (os == "linux") and not webrender and not debug and (processor == "x86_64"): ["TIMEOUT", "PASS"]
-      if (os == "win") and webrender and not fission and not swgl: ["TIMEOUT", "PASS"]
-      if (os == "win") and webrender and not fission and swgl: ["PASS", "TIMEOUT"]
+      if (os == "win") and webrender and not debug and not fission: ["TIMEOUT", "PASS"]
+      if (os == "win") and webrender and not debug and fission: ["TIMEOUT", "PASS"]
       if (os == "win") and not webrender and not debug and (processor == "x86_64"): ["TIMEOUT", "PASS"]
+      if (os == "linux") and not webrender and not debug: ["TIMEOUT", "PASS"]
       if (os == "android") and debug and not webrender: ["TIMEOUT", "PASS"]
-      if (os == "win") and webrender and fission: ["TIMEOUT", "PASS"]
+      if (os == "win") and webrender and debug: ["TIMEOUT", "PASS"]
       if (os == "mac") and debug: ["TIMEOUT", "PASS"]
       TIMEOUT
 